server/src/test/java/org/elasticsearch/persistent/PersistentTasksClusterServiceTests.java

```diff
@@ -670,15 +670,15 @@ public void testReassignOnlyOnce() throws Exception {
         Thread t2 = new Thread(service::reassignPersistentTasks);
         try {
             t1.start();
-            t2.start();
             // Make sure we have at least one reassign check before we count down the latch
             assertBusy(
                 () -> verify(recheckTestClusterService, atLeastOnce()).submitStateUpdateTask(eq("reassign persistent tasks"), any())
```

Comment:
Should this now be exactly once instead of at least once?  If not, please add a comment to say why there might be more than one.

Reply:
@droberts195 the `atLeastOnce` is because these `verify` checks are also present after the test has ran. If I did `1` time only, the test would fail because there is a valid execution later (when both have finished executing making sure the atomicboolean is unset).